hl7/4.2/modules/ROOT/pages/hl7-connector-examples.adoc

```diff
@@ -1,27 +1,25 @@
 = HL7 EDI - Examples - Mule 4
 
-Use these examples to see how to customize a schema, configure the Message Header (MSH) application and facility identification for you and your trading partner, view a test flow with an HTTP Listener that initiates a flow.
+Use the following examples to see how to customize a schema, configure the Message Header (MSH) application and facility identification for you and your trading partner, and view a test flow with an HTTP Listener that initiates a flow.
 
 === Simplify a Schema Using Example Messages
 
 The HL7 standard definitions are very complex, with segments often
-having twenty or more components and many of the components composites
-which themselves are broken down into many subcomponents. This can
+having twenty or more components of which many of themselves are broken down into subcomponents. This component nesting can
```

Comment:
Awkward sentence. I suggest "having twenty or more components, many of which are broken down into subcomponents.